# Max error of amcl pose to start the algo making new pose candidate
max_error_x: 0.5
max_error_y: 0.5
max_error_theta: 0.5

# Max deviation to consider the data as a stable data
max_dev_x: 0.2
max_dev_y: 0.2
max_dev_theta: 0.3

# Number of comparison at stability checker
num_holding: 5

# Range for aruco saver
correction_range: 2.5

# color_optical_frame topic camera_color_optical_frame camera_rgb_optical_frame

optical_frame : "camera_color_optical_frame" 